numpy/add_newdocs.py

```diff
@@ -374,13 +377,50 @@ def luf(lamdaexpr, *args, **kwargs):
             while not it.finished:
                 it[0] = lamdaexpr(*it[1:])
                 it.iternext()
+            it.close()
             return it.operands[0]
 
         >>> a = np.arange(5)
         >>> b = np.ones(5)
         >>> luf(lambda i,j:i*i + j/2, a, b)
         array([  0.5,   1.5,   4.5,   9.5,  16.5])
 
+    If operand flags `"writeonly"` or `"readwrite"` are used the operands may
+    be views into the original data with the WRITEBACKIFCOPY flag. In this case
+    it is best practice to use the nditer as a context manager. The temporary
+    data will be written back to the original data when the `` __exit__``
+    function is called but not before::
+
+        >>> a = np.arange(6, dtype='i4')[::-2]
+        >>> with nditer(a, [],
+        ...        [['writeonly', 'updateifcopy']],
+        ...        casting='unsafe',
+        ...        op_dtypes=[np.dtype('f4')]) as i:
+        ...    i.operands[0][:] = [-1, -2, -3]
+        ...    # a still unchanged here
+        >>> a
+        array([-1, -2, -3])
+
+    Additionally, helper function ``close`` can be used to force resolution of
+    the temporary data::
+
+        >>> a = np.arange(6, dtype='i4')[::-2]
+        >>> it = nditer(a, [], [['writeonly', 'updateifcopy']],
+        ...        casting='unsafe',op_dtypes=[np.dtype('f4')])
+        >>> x = it.operands[0][:] = [-1, -2, -3]
+        >>> it.close()
+        >>> a
+        array([5, 3, 1])
```

Comment:
This part about `close` should be removed

Reply:
fixed